content/index.rst

```diff
@@ -23,19 +23,19 @@ branching.
 
 .. prereq::
 
-   -  A reasonably recent version of Git is installed. For installation
-      instructions see `the CodeRefinery installation
+   -  A reasonably recent version of **Git is installed**
```

Comment:
Do you know what our oldest requirement is?

* `git switch` is 2.23 (2019-08-16)
* `init.defaultBranch` is 2.28 (2020-07-27)

I don't think we do the second - do we do the first?

Reply:
We haven't concluded on changing either yet so we postpone to another year.

Reply:
So in other words we don't require reasonable recent version but it should work with almost any ancient version.